pages/about_us.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="../styles/style.css">
+        <link rel="stylesheet" href="../styles/about_us.css">
+        <link rel="icon" type="image/x-icon" href="../images/logo.gif">
+        <title>About Us</title>
+    </head>
+    <body>
+        <header>
+            <nav>
+                <a href="../index.html">Home</a>
+                <a href="facts.html">Fun Facts</a>
+                <a href="gallery.html">Gallery</a>
+                <a href="#" class="active">About Us</a>
+            </nav>
+        </header>
+        <main>
+            <section>
+                <section class="logo">
+                    <img src="../images/logo.gif" alt="Coffee Lovers logo">
+                </section>
+                <h1>About Us</h1>
+                <h2>From bean to screen: meet your coffee loving team</h2>
+                <section class="container">
+                    <section class="team-member">
+                        <img src="../images/pic_aleida.jpeg" alt="Coffee lovers team member Aleida">
+                        <section>
```

Comment:
We usually expect `section` tags to have headers, what could be the most appropriate tag to use here?